[2021-02-28 09:16:44,899] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_sample_file3 2021-02-28T08:16:29.182049+00:00 [queued]>
[2021-02-28 09:16:44,904] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_sample_file3 2021-02-28T08:16:29.182049+00:00 [queued]>
[2021-02-28 09:16:44,904] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:16:44,904] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-28 09:16:44,904] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:16:44,909] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): create_sample_file3> on 2021-02-28T08:16:29.182049+00:00
[2021-02-28 09:16:44,911] {standard_task_runner.py:52} INFO - Started process 35205 to run task
[2021-02-28 09:16:44,918] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'create_sample_file3', '2021-02-28T08:16:29.182049+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp6f7x8bi4', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmpe3rregig']
[2021-02-28 09:16:44,919] {standard_task_runner.py:77} INFO - Job 10: Subtask create_sample_file3
[2021-02-28 09:16:44,945] {logging_mixin.py:104} INFO - Running <TaskInstance: my_first_dag.create_sample_file3 2021-02-28T08:16:29.182049+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 09:16:44,967] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=create_sample_file3
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T08:16:29.182049+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T08:16:29.182049+00:00
[2021-02-28 09:16:44,968] {python.py:118} INFO - Done. Returned value was: None
[2021-02-28 09:16:44,971] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=create_sample_file3, execution_date=20210228T081629, start_date=20210228T081644, end_date=20210228T081644
[2021-02-28 09:16:44,980] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-28 09:16:45,018] {local_task_job.py:146} INFO - Task exited with return code 0
